src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -506,52 +510,51 @@ def launch_fluent(
         value.
     cleanup_on_exit : bool, optional
         Whether to shut down the connected Fluent session when PyFluent is
-        exited or the ``exit()`` method is called on the session instance.
-        The default is ``True``.
+        exited, or the ``exit()`` method is called on the session instance,
+        or if the session instance becomes unreferenced. The default is ``True``.
     start_transcript : bool, optional
         Whether to start streaming the Fluent transcript in the client. The
         default is ``True``. You can stop and start the streaming of the
-        Fluent transcript subsequently via method calls on the session object.
+        Fluent transcript subsequently via the method calls, ``start_transcript()``
+        and ``stop_transcript()`` on the session object.
     show_gui : bool, optional
-        Whether to display the Fluent GUI when ``start_instance``
-        is set to ''True``. The default is ``None`` so that explicit
-        ``False`` settings can be detected. This is because you can use
-        also use the environment variable ``PYFLUENT_SHOW_SERVER_GUI=<0 or 1>``
-        to set this parameter. The ``show-gui`` parameter overrides the
-        PYFLUENT_SHOW_SERVER_GUI environment variable. For example, if
-        PYFLUENT_SHOW_SERVER_GUI is set to ``1`` and the ``show-gui``
-        parameter is set to ``False``, the GUI is hidden.
+        Whether to display the Fluent GUI, only when ``start_instance``
+        is set to ``True``. The default is ``None``, which does not
+        cause the GUI to be shown. If a value of ``False`` is
+        not explicitly provided, the GUI will also be shown if
+        the environment variable ``PYFLUENT_SHOW_SERVER_GUI`` is set to 1.
     case_filepath : str, optional
-        If provided, reads a fluent case file and sets the required settings
-        in the fluent session
+        If provided, the case file at ``case_filepath`` is read into the Fluent session.
     case_data_filepath : str, optional
-        If provided, reads a fluent case and data file and sets the required settings
-        in the fluent session
+        If provided, the case and data files at ``case_data_filepath`` are read into the Fluent session.
     lightweight_mode: bool, optional
-        Whether to use light io mode for reading case via ``case_filepath`` parameter.
-        This parameter is used only when ``case_filepath`` is provided. The default is
-        ``False``.
+        Whether to read only the lightweight settings from any specified Fluent case file, ignoring the mesh.
```

Comment:
The lightwight_mode reads the lightweight settings in the current solver session. The mesh will be read in a background solver session which will replace the current solver session after applying any change in the lightweight settings made by user in the current session.

Reply:
@mkundu1 it seems that your comment above captures better than either of the docstrings. I will revisit this now.